### PR TITLE
Readd Japanese and French translations to error.go

### PR DESCRIPTION
**Pull request by ppebb**
Readds the changes in https://github.com/Retro-Rewind-Team/wfc-server/pull/5. Fixes syntax and formatting as well as inconsistent newlines.

Someone needs to check over the newlines to make sure they are consistent. I've checked three times but I can't be sure it's consistent.

### CODE REVIEW

File: gpcm/error.go [hunk 1]
```diff
@@ -250,6 +322,16 @@ var (
 				"Retro WFC.\n" +
 				"\n" +
 				"Error Code: %[1]d",
+			LangJapanese: "" +
+				"Retro WFCから キックされました\n" +
+				"公式レースから BANされました\n" +
+				"\n" +
+				"エラーコード： %[1]d",
```
Comment (ppebb) [inline gpcm/error.go:329]: @Eppeu this appears to mention bans, but this is for the WWFCMsgKickedGeneric message, so it should say the player is kicked.

**Comment by chairclr** [comment]
lgtm

**Review by TheStachelfisch — APPROVED** [review]
lgtm

**Comment by Smug-B** [comment]
LGTM (Let's Get This Merged!)

File: gpcm/error.go (hunk 1 above)
Comment (Eppeu) [reply gpcm/error.go:329]: Hey, sorry about the wait for this, I can't work that much, you only need to remove:
 "公式レースから BANされました\n" +
 It was a mistake from my friend when we made it

**Comment by ppebb** [comment]
lgmt (let's get this merged)

**Pull request merged**